apps/frontend/public/static/export/template.html

```diff
@@ -1,271 +1,637 @@
-                          </thead>
-                          <tbody id="controlTestResultTableBody"> </tbody>
-                        </table>
-                      </div>
-                      <hr />
-                      <div class="col">
-                        <h5>Control Details</h5>
-                        <table class="table table-striped table-hover" id="controlDetailsTable"
-                          aria-describedby="Control details">
-                          <thead>
-                            <tr>
-                              <th scope="col" style="width: 20%">Name</th>
-                              <th scope="col">Value</th>
-                            </tr>
-                          </thead>
-                          <tbody> {{#details}}
-                            <tr>
-                              <td>{{name}}</td>
-                              <td>{{value}}</td>
-                            </tr> {{/details}} </tbody>
-                        </table>
-                      </div>
-                      {{#showCode}}
-                      <hr />
-                      <div class="col">
-                        <pre>
-<code>{{full_code}}</code>
-                        </pre>
-                      </div> {{/showCode}}
-                    </div>
+        </div>
+
+        <!-- Inner accordion -->
+        <!-- Accordion body def -->
+        <div
+          id="collapseable-{{fileID}}"
+          class="multi-collapse-outer my-4"
+          data-te-collapse-item
+          data-te-collapse-show
+        >
+          <!-- Control header -->
+          <div class="mx-4 rounded-t-xl border border-gray-300 bg-gray-100">
+            <div class="mx-6 my-4 flex w-full shrink">
+              <!-- Status indicator -->
+              <!-- Headers eventually collapse under status column -->
+              <span class="grid w-36 grid-cols-1 sm:w-40">
+                <!-- ID -->
+                <span class="sm:hidden">ID</span>
+                <hr class="h-1 w-32 bg-gray-300 sm:hidden" />
+                <!-- Status indicator -->
+                <span>Status</span>
+                <hr class="h-1 w-32 bg-gray-300 lg:hidden" />
+                <!-- Severity indicator -->
+                <span class="lg:hidden">Severity</span>
+                <hr class="h-1 w-32 bg-gray-300 lg:hidden" />
+                <!-- Control tags -->
+                <span class="lg:hidden">800-53 Controls & CCIs</span>
+              </span>
+              <!-- ID -->
+              <span class="hidden w-28 pl-4 sm:block lg:w-32">ID</span>
+              <!-- Severity indicator -->
+              <span class="hidden w-28 pl-3 lg:block">Severity</span>
+              <!-- Title -->
+              <span class="w-1/2 pl-8 sm:block lg:pl-9">Title</span>
+              <!-- Control tags -->
+              <span class="hidden lg:block lg:w-36 xl:w-52"
+                >800-53 Controls & CCIs</span
```

Comment:
please find/fix all instances of

```
<tag
  >content inside</tag
>
```

it should be
```
<tag>
  content inside
</tag>
```

or a one liner if it fits according to the linter